index.html

```diff
@@ -1,43 +1,264 @@
+        A daily selection of privately personalized reads; no accounts or
+        sign-ups required.
+      </p>
+      <ul class="btns">
+        <li>
+          <p>html</p>
         </li>
         <li>
-            <a href=""><i class="fab fa-github"></i></a>
+          <p>css</p>
         </li>
         <li>
-            <a href=""><i class="fab fa-linkedin-in"></i></a>
+          <p>javascript</p>
         </li>
+      </ul>
+      <a href="#" class="view"><p>See Project</p></a>
+    </div>
+
+    <!-- Multi-Post-Stories-1 -->
+
+<div class="wrapper-3">
+    <div class="snapshoot-2"></div>
+    <h2 class="multi-post-stories">
+        Multi-Post <br />
+        Stories
+      </h2>
+      <div class="list">
+        <p class="canopy">CANOPY</p>
+        <a href="" class="date"><p>Back End Dev</p></a>
+        <a href="" class="date"><p>2015</p></a>
+      </div>
+      <p class="content-2">
+        A daily selection of privately personalized reads; no accounts or
+        sign-ups required.
+      </p>
+      <ul class="btns">
         <li>
-            <a href=""><i class="fab fa-medium-m"></i></a>
+          <p>html</p>
         </li>
         <li>
-            <a href=""><i class="far fa-hand-peace"></i></a>
+          <p>css</p>
+        </li>
+        <li>
+          <p>javascript</p>
+        </li>
+      </ul>
+      <a href="#" class="view"><p>See Project</p></a>
+
+</div>
+    <!-- Tonic-2 -->
+
+    <div class="wrapper-2">
+      <div class="snapshoot-3"></div>
+      <h2 class="tonic">Tonic</h2>
+
+      <div class="list">
+        <p class="canopy">CANOPY</p>
+        <a href="" class="date"><p>Back End Dev</p></a>
+        <a href="" class="date"><p>2015</p></a>
+      </div>
+      <p class="content-2">
+        A daily selection of privately personalized reads; no accounts or
+        sign-ups required.
+      </p>
+      <ul class="btns">
+        <li>
+          <p>html</p>
+        </li>
+        <li>
+          <p>css</p>
+        </li>
+        <li>
+          <p>javascript</p>
+        </li>
+      </ul>
+      <a href="#" class="view"><p>See Project</p></a>
+    </div>
+
+    <!-- Multi-Post-Stories-2 -->
+
+    <div class="wrapper-3">
+      <div class="snapshoot-4"></div>
+      <h2 class="multi-post-stories">
+        Multi-Post <br />
+        Stories
+      </h2>
+
+      <div class="list">
+        <p class="canopy">CANOPY</p>
+        <a href="" class="date"><p>Back End Dev</p></a>
+        <a href="" class="date"><p>2015</p></a>
+      </div>
+      <p class="content-2">
+        A daily selection of privately personalized reads; no accounts or
+        sign-ups required.
+      </p>
+      <ul class="btns">
+        <li>
+          <p>html</p>
+        </li>
+        <li>
+          <p>css</p>
+        </li>
+        <li>
+          <p>javascript</p>
+        </li>
+      </ul>
+      <a href="#" class="view"><p>See Project</p></a>
+    </div>
+
+    <!-- About Myself -->
+
+    <div class="wrapper-about">
```

Comment:
- [ ] Make sure to use `section` tag instead of `div` in the About section, which will improve your project to look more semantic and organized, you can read about this in this [link](https://www.w3schools.com/html/html5_semantic_elements.asp)